addons/core/XEH_PREP.sqf

```diff
@@ -177,6 +179,8 @@ PREP_SUB(hint,showRadioVolume);
 PREP(setChannelFrequency);
 PREP(setRadioOwner);
 PREP(setLrSpeakers);
+PREP(setSwHalfDuplexOverride);
+PREP(setLrHalfDuplexOverride);
```

Comment:
Yeah, I'll see if I can make it work in one function. If it gets too much work or hassle of checking which type we're handling, I might just leave it separated.
Will try to merge it into one function though!